### PR TITLE
[482] Adds business name to the dashboard endpoint

### PR DESCRIPTION
**Pull request by katelovescode**
Adds business name to the dashboard endpoint, adds an admin user to seeds for local manual testing

# 💅🏼 What issue does this fix?
#482 

## 🍂 Before You Submit

* [X] Did you write tests?
* [X] Did you run `bundle exec rspec` from the root?
* [X] Did you run `bundle exec rails rswag` from the root?
* [X] Did you run `bundle exec rubocop` from the root?

## 🧳 Steps to test

- login via Postman
- use the token from the Authorization header in the login response in your request Authorization header as a bearer token
- hit the new endpoint: `/api/v1/case_list_for_dashboard`
- you should see the following response format, if the user you logged in with has any children associated with it (or if they are an admin user):
```
[
    {
        "full_name": "Maria Baca",
        "business": {
            "name": "Happy Seedlings Childcare"
        },
        "approvals": [
            {
                "case_number": "5302337801"
            }
        ]
    },
    {
        "full_name": "K'Shawn Henderson",
        "business": {
            "name": "Happy Seedlings Childcare"
        },
        "approvals": [
            {
                "case_number": "3124809659"
            }
        ]
    }
]
```

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -26,6 +26,20 @@ def puts_records_in_db(klass)
 
 # ---------------------------------------------
 
+@user_admin = User.where(email: 'admin@test.com').first_or_create!(
+  full_name: 'Admin User',
+  greeting_name: 'Addie',
+  language: 'spanish',
+  opt_in_email: false,
+  opt_in_text: false,
+  organization: 'Pie for Providers',
```
Comment (weedySeaDragon) [inline db/seeds.rb:35]: We're using `ENV` to get the email and password for the 1 user created, should we use `ENV` here, too?
- or only use `ENV` for the admin
- or not use `ENV` at all?

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]
Should this return only _active_ businesses or all of them?
And if all businesses, should this also return the _active_ attribute?

**Comment by katelovescode** [comment]
Great question. We'll need a check on children that if their business is not active they must be turned inactive or assigned to an active business. Then this dashboard query should only return active children. I'll add that logic in.

File: db/seeds.rb (hunk 1 above)
Comment (katelovescode) [reply db/seeds.rb:35]: I'm gonna lean towards no ENVs, I think it makes it more confusing right now.

**Pull request merged**